KsApi/MockService.swift

```diff
@@ -583,13 +580,14 @@
     }
 
     internal func changeCurrency(input _: ChangeCurrencyInput) ->
-      SignalProducer<GraphMutationEmptyResponseEnvelope, GraphError> {
-      if let response = self.changeCurrencyResponse {
+    SignalProducer<UpdateUserEnvelope, ErrorEnvelope> {
```

Comment:
Yea good point, I can technically use the `GraphMutationEmptyResponseEnvelope` (or create a new `EmptyResponseEnvelope`) but my thinking is that `UpdateUserEnvelope` acts kind of like an empty response. It returns a dictionary like this:
```
    let dict: [String: Any?] = [
      "updateUserAccount": [
        "clientMutationId": nil
      ]
    ]
```
And the absence of the `updateUserAccount` value in the return dictionary means the mutation failed. So that's how the `UpdateUserEnvelope` works.

Reply:
How were we detecting errors beforehand when we were calling graph and expecting a `GraphMutationEmptyResponseEnvelope`?

Reply:
Yea so the `UpdateAccountEnvelope.from(data)` will call up to the `static func from(_ data: GraphAPI.UpdateUserAccountMutation.Data) -> UpdateUserEnvelope?` method which will check if the `data` has an `updateUserAccount` property. That property is only there if the `GraphAPI` parses the json and finds the dictionary value for key `updateUserAccount`.

line `2770` of `GraphAPI`, basically `ResultMap` is just a `[String : Any?]` dictionary.

Reply:
Ok, removed `UpdateUserEnvelope`, adapters and tests. Now using `EmptyResponseEnvelope`. Tested all related calls `changePassword`, `changeEmail`, `createPassword` and `changeCurrency` they work as expected.